subsys/shell/Kconfig.backends

```diff
@@ -21,5 +21,12 @@ config SHELL_BACKEND_SERIAL
 	help
 	  Enable serial backends.
 
+config SHELL_BACKEND_DUMMY
+	bool "Enable dummy backend."
+	select SERIAL
+	help
+	  Enable dummy backend which can be used to test commands with no need
```

Comment:
`can be used to test commands` I would change test here to execute

Reply:
done